languages/cpp/exercises/concept/numbers/.meta/design.md

```diff
@@ -0,0 +1,26 @@
+## Goal
+
+The goal of this exercise is to teach the student how to work with integer and floating point numbers in C++.
+
+## Learning objectives
+
+- Know the different types of integer and floating point numbers.
+- Know integer type modifiers.
+- Know of basic operators such as multiplication, comparison and equality.
+- Know how to conditionally execute code using an `if` statement.
```

Comment:
I didn't see anything about `if` statements. Did I miss it?

Reply:
The simplest way to do the rate map is with an `if` statement. Should I add this to references?

Reply:
> I didn't see anything about if statements. Did I miss it?

Great observation @siebenschlaefer! One of the concepts this exercise introduces is conditional execution of code using if-statements. As such, the introduction.md (and after.md) should have a section on `if` statements.